src/platform_impl/windows/event_loop/runner.rs

```diff
@@ -33,7 +33,7 @@ pub(crate) struct EventLoopRunner<T: 'static> {
     event_handler: Cell<Option<Box<dyn FnMut(Event<'_, T>, &mut ControlFlow)>>>,
     event_buffer: RefCell<VecDeque<BufferedEvent<T>>>,
 
-    owned_windows: Cell<HashSet<HWND>>,
+    owned_windows: Cell<HashSet<isize>>,
```

Comment:
@kennykerr I'm not entirely sure why this needs to be `isize` (probably `Hash` not being implemented on `HWND`?) but it would be nice to support this use case

Reply:
Yes, its on my radar: https://github.com/microsoft/windows-rs/issues/589

Reply:
This is fixed with 0.29 since HWND is now just defined as:
```rust
pub type HWND = isize;
```
Thanks @kennykerr!